src/components/PhotoFrame.tsx

```diff
@@ -392,17 +339,17 @@ const PhotoFrame = ({
                 files[index].html = `
                 <div class="video-loading">
                     <img src="${files[index].msrc}" />
-                    <div class="download-message">
+                    <div class="download-message" data-id="${files[index].id}">
                         ${constants.VIDEO_PLAYBACK_FAILED_DOWNLOAD_INSTEAD}
                         <button class = "btn btn-outline-success">Download</button>
```

Comment:
wouldn't attaching an onClick callback here itself be an easier solution. Did it not work for some reason?

Reply:
It showed an error that you can't directly add Javascript in HTML (I don't remember the exact error, will let you know).